docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -456,6 +456,40 @@ Here are two concrete examples:
 1. The user cleans the year, the rendered value is `01/01/YYYY`, `onChange` is fired with `null`.
 2. The user enters a new year, the rendered value is `01/01/2026`, `onChange` is fired with the new date.
 
+### ⏩ Deprecate the `disableOpenPicker` prop
+
+The `disableOpenPicker` prop has been deprecated on all the picker components and will be removed in the next major release (v9.0.0).
+If you only want to allow editing through the field, you can use the field component directly:
```

Comment:
WDYT about additionally linking to the doc for anyone still unaware of the concept? 🤔 
```suggestion
If you only want to allow editing through the field, you can use the [field component](/x/react-date-pickers/fields/) directly:
```